### PR TITLE
UI: New Logos, Updated StyledButton, StyledCheckbox, Component-Update for DeleteBankAccount

### PR DESCRIPTION
**Pull request by niklasdfx**

### CODE REVIEW

File: src/stories/form/StyledBankAccountListItem.tsx [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
-      <div className="flex flex-col justify-center h-[42px]">
-        {isLabelAvailable && <p className="text-dfxGray-800 text-xs">{bankAccount.label}</p>}
-        <p className="text-dfxBlue-800">{Utils.formatIban(bankAccount.iban)}</p>
+      <div className="hidden group-hover:block">
+        <StyledIconButton
+          icon={IconVariant.DELETE_OUTLINE}
+          color={IconColors.RED}
+          size={IconSizes.LG}
+          onClick={onDelete}
+        />
```
Comment (Krysh90) [inline src/stories/form/StyledBankAccountListItem.tsx:30]: We will need to deactivate / hide this button for now.
API is ready, but exchange hasn't implemented the functionality for it yet.
If we need to release it early due to asset icon changes, we need to hide it for now

**Review by Krysh90 — APPROVED** [review]

**Comment by Krysh90** [comment]
Generally speaking, if we have other changes, which are directly visible and where the logic behind is missing, we would need to hide it until the logic is built.

File: src/stories/form/StyledBankAccountListItem.tsx (hunk 1 above)
Comment (niklasdfx) [reply src/stories/form/StyledBankAccountListItem.tsx:30]: yes, i thought about it this morning, that i wanted to discuss with you two. if today is just about the token, i should comment that stuff out ...

File: src/stories/layout-helpers/StyledVerticalStack.tsx [hunk 2]
```diff
@@ -5,18 +5,18 @@ export interface StyledVerticalStackProps extends PropsWithChildren {
...
-export default function StyledVerticalStack({ children, gap = 0, marginY, marginX }: StyledVerticalStackProps) {
-  let mY: string | undefined;
-  let mX: string | undefined;
-
+export default function StyledVerticalStack({ children, gap = 0, marginY, marginX, center }: StyledVerticalStackProps) {
   const spacing = convertToRem(gap);
-  marginY !== undefined ? (mY = convertToRem(marginY)) : (mY = '0');
-  marginX !== undefined ? (mX = convertToRem(marginX)) : (mX = '0');
+  const mY = marginY !== undefined ? convertToRem(marginY) : '0';
+  const mX = marginX !== undefined ? convertToRem(marginX) : '0';
```
Comment (davidleomay) [inline src/stories/layout-helpers/StyledVerticalStack.tsx:14]: No strict equality for comparison to `undefined`, e.g.:
```ts
marginY != undefined
```

File: src/stories/StyledButton.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import StyledLoadingSpinner, { SpinnerSizes, SpinnerVariant } from './StyledLoadingSpinner';
+import DfxIcon, { IconColors, IconSizes, IconVariant } from './DfxIcon';
 
 export enum StyledButtonSizes {
```
Comment (davidleomay) [inline src/stories/StyledButton.tsx:4]: Not related to this PR, but enum names should be singular (e.g. `StyledButtonSize`). Same for `StyledButtonColor`.

File: src/stories/StyledCheckboxRow.tsx [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+export interface StyledCheckboxRowProps extends PropsWithChildren {
+  darkTheme?: boolean;
+  onChange: (isChecked: boolean) => any;
+  isChecked?: boolean;
+  withBackground?: boolean;
+  centered?: boolean;
+  textSize?: StyledCheckboxRowTextSizes;
+}
+
+export enum StyledCheckboxRowTextSizes {
```
Comment (davidleomay) [inline src/stories/StyledCheckboxRow.tsx:13]: Same here 😊

**Review by davidleomay — COMMENTED** [review]
Wait for integration by @Krysh90.

**Review by davidleomay — APPROVED** [review]

**Pull request merged**